subsys/bluetooth/services/latency.c

```diff
@@ -0,0 +1,159 @@
+/*
```

Comment:
This seems to be both Service and Client in one file. We usually have one file for each. I'd suggest to split it.

Reply:
Because there is only one special case in function test_run() on the master, we tend to keep it simple as same as throughput. Then, it won't introduce too much duplicate code,

Reply:
Alright, I misunderstood it in the first place. It sounds good to split the latency service to be reusable.